src/Sensor/CpuMeasurement.java

```diff
@@ -6,10 +6,11 @@
 
 public class CpuMeasurement extends Measurement{
 
+    final static String MAX_MEASURE_VALUE = "100";
```

Comment:
Wouldn't `MAX_PERCENTAGE_MEASURE_VALUE` be more readable? If it's not percentage just ignore this comment.